### PR TITLE
Update neetocommons/pure imports to neeto-cist

### LINKED ISSUE #920: Migrate imports from `neetocommons/pure` to `neeto-cist`
ref: https://github.com/bigbinary/neeto-commons-frontend/issues/825

### PR DESCRIPTION
**Pull request by AkhilXavier95**
- Fixes #920 

**Description**
Migrated imports from `neetocommons/pure` to `neeto-cist`
  
**Checklist**

~- [ ] I have made corresponding changes to the documentation.~
~- [ ] I have updated the types definition of modified exports.~
~- [ ] I have verified the functionality in some of the neeto web-apps.~
- [x] I have added the necessary label (patch/minor/major - If package publish
      is required).

### CODE REVIEW

**Comment by AkhilXavier95** [comment]
@ajmaln _a please review this PR.

File: resolve.js [hunk 1]
```diff
@@ -3,5 +3,6 @@ const path = require("path");
 module.exports = {
   alias: {
     common: path.resolve(__dirname, "./src/common"),
+    neetocist: "@bigbinary/neeto-cist",
```
Comment (ajmaln) [inline resolve.js:6]: This can be removed, just update `neeto-commons-frontend` to the latest version

**Review by ajmaln — APPROVED** [review]
@AkhilXavier95 _a Please check, rest LGTM

File: package.json [hunk 2]
```diff
@@ -33,7 +33,8 @@
     "@bigbinary/babel-preset-neeto": "^1.0.4",
     "@bigbinary/eslint-plugin-neeto": "^1.0.54",
     "@bigbinary/neeto-audit-frontend": "2.0.1",
-    "@bigbinary/neeto-commons-frontend": "^2.0.128",
+    "@bigbinary/neeto-cist": "^1.0.2",
+    "@bigbinary/neeto-commons-frontend": "^2.1.27",
```
Comment (ajmaln) [inline package.json:37]: Downgrade to `2.1.23`

File: package.json [hunk 3]
```diff
@@ -33,7 +33,8 @@
     "@bigbinary/babel-preset-neeto": "^1.0.4",
     "@bigbinary/eslint-plugin-neeto": "^1.0.54",
     "@bigbinary/neeto-audit-frontend": "2.0.1",
-    "@bigbinary/neeto-commons-frontend": "^2.0.128",
+    "@bigbinary/neeto-cist": "^1.0.2",
```
Comment (ajmaln) [inline package.json:36]: add to peerDeps

**Review by ajmaln — COMMENTED** [review]
@AkhilXavier95 _a

**Comment by AkhilXavier95** [comment]
@ajmaln _a please review

**Review by ajmaln — APPROVED** [review]

**Pull request merged**